packages/client-core/src/components/Glass/ToolbarAndSidebar.tsx

```diff
@@ -195,6 +195,8 @@ const contentStyles = `
   
   lg:px-14
   lg:pt-12
+
+  w-full
```

Comment:
The `absolute` and `inset-0` should be making this element full height and width of the parent already. Also I think `w-full` wouldnt work as expected cus its `absolute`